packages/api/core/src/api/start.ts

```diff
@@ -52,7 +52,7 @@ export default async ({
     forgeConfig.electronRebuildConfig,
   );
 
-  await runHook(forgeConfig, 'generateAssets');
+  await runHook(forgeConfig, 'generateAssets', process.platform, process.arch);
```

Comment:
This is actually incorrect, because of use cases like [WSL](https://www.electronforge.io/guides/developing-with-wsl)...

I'm not sure how we can fix this problem, actually.

Reply:
That's what it currently passes to electron rebuild a few lines above this:
https://github.com/electron-userland/electron-forge/blob/94b0f962ad756edcd282826bdc304b14744fd6e6/packages/api/core/src/api/start.ts#L50-L51